api/src/main/java/org/pmiops/workbench/cohorts/CohortService.java

```diff
@@ -31,7 +31,7 @@ public CohortService(
   }
 
   public List<Cohort> findAll(List<Long> cohortIds) {
-    return ((List<DbCohort>) cohortDao.findAll(cohortIds))
+    return ((List<DbCohort>) cohortDao.findAllById(cohortIds))
```

Comment:
this is a sketchy cast - we should remove it as well as the cast in ConceptService

Reply:
right, that does seems not good. Changed